File: app/src/main/java/com/nelyan/adapter/NotificationAdapter.java
package com.nelyan.adapter;

import android.content.Context;
import android.content.Intent;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.RelativeLayout;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.nelyan.R;

public class NotificationAdapter extends RecyclerView.Adapter<NotificationAdapter.RecyclerViewHolder> {
    Context context;
    LayoutInflater inflater;
    RelativeLayout rl_1;
    public class RecyclerViewHolder extends RecyclerView.ViewHolder {
        public RecyclerViewHolder(View view) {
            super(view);

        }
    }

    public NotificationAdapter(Context context) {
        this.context = context;
        inflater = LayoutInflater.from(context);
    }

    @Override
    public NotificationAdapter.RecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
        View v = inflater.inflate(R.layout.list_noti, parent, false);

        NotificationAdapter.RecyclerViewHolder viewHolder = new NotificationAdapter.RecyclerViewHolder(v);
        return viewHolder;
    }
    @Override
    public void onBindViewHolder(@NonNull NotificationAdapter.RecyclerViewHolder holder, final int i) {

    }
    @Override
    public int getItemCount() {
        return 5;
    }


}

File: app/src/main/java/com/nelyan_live/adapter/NotificationAdapter.kt
package com.nelyan_live.adapter

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.RelativeLayout
import androidx.recyclerview.widget.RecyclerView
import com.nelyan_live.R

class NotificationAdapter(var context: Context) : RecyclerView.Adapter<NotificationAdapter.RecyclerViewHolder>() {
    var inflater: LayoutInflater
    var rl_1: RelativeLayout? = null

    inner class RecyclerViewHolder(view: View?) : RecyclerView.ViewHolder(view!!)

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerViewHolder {
        val v = inflater.inflate(R.layout.list_noti, parent, false)
        return RecyclerViewHolder(v)
    }

    override fun onBindViewHolder(holder: RecyclerViewHolder, i: Int) {}
    override fun getItemCount(): Int {
        return 5
    }

    init {
        inflater = LayoutInflater.from(context)
    }
}